README.rst

```diff
@@ -16,6 +16,20 @@ Features
 
 * TODO
 
+PyPI Package Release
+--------------------
+- This reposiroty publishes package on PyPI, when you update this repository make sure to create a new release so latest changes should be published on PyPI and become available for your use.
+- Bump the version `utils version`_ to match the release version, once pull reuqest with updated version is merged into master. Create a new release from GitHub Web UI. You can refer this document on how to `How to create GitHub release`_
```

Comment:
nits:  "reuqest" => "request". 

I'm also not sure I understand the steps here.   Is it something like this? "Bump the version to match the version that will be released.  Once the pull request with the updated version is merged into master, create a new release from GitHub Web UI, using the same version number."

I was also confused about the last line:  is "this document" referring to the README or the one you point to?   I assume the latter.   It might be a bit clearer if it read: "You can refer to `How to create a GitHub release` _ for more information on this process.